[2;32m2016-03-30 10:30:49,705 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2016-03-30 10:30:50,705 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2016-03-30 10:30:50,705 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2016-03-30 10:30:50,705 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2016-03-30 10:30:51,184 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2016-03-30 10:30:51,248 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2016-03-30 10:30:51,279 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2016-03-30 10:30:51,310 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2016-03-30 10:30:51,310 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2016-03-30 10:30:51,357 - INFO  [main:ServerBootstrap@129] - Application PID:	5876
[m[2;35m2016-03-30 10:32:41,362 - WARN  [Thread-10:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :84505, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 10:33:00,691 - WARN  [Thread-6:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :14441, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 10:33:06,475 - WARN  [Thread-2:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :20, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 10:33:07,646 - WARN  [Thread-23:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :858, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryFilterBySpid
[m[2;35m2016-03-30 10:33:07,841 - WARN  [Thread-4:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :12, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[2;35m2016-03-30 10:42:48,959 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :265850, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[7;31m2016-03-30 11:05:42,819 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2016-03-30 11:05:42,840 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2016-03-30 11:05:44,844 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[2;32m2016-03-30 17:40:22,908 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2016-03-30 17:40:23,578 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2016-03-30 17:40:23,578 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2016-03-30 17:40:23,578 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2016-03-30 17:40:24,050 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2016-03-30 17:40:24,168 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2016-03-30 17:40:24,199 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2016-03-30 17:40:24,215 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2016-03-30 17:40:24,248 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2016-03-30 17:40:24,335 - INFO  [main:ServerBootstrap@129] - Application PID:	6292
[m[2;35m2016-03-30 17:41:20,268 - WARN  [Thread-33:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :3486, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:41:20,366 - WARN  [Thread-23:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :8, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:43:21,200 - WARN  [Thread-14:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:43:21,219 - WARN  [Thread-35:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :8, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:43:21,475 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :13, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2016-03-30 17:43:36,133 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2016-03-30 17:43:36,174 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2016-03-30 17:43:38,181 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[7;31m2016-03-30 17:43:39,207 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 17:43:39,207 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 17:43:39,207 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2016-03-30 17:43:41,223 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 2 times
[m[2;35m2016-03-30 17:43:41,223 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2016-03-30 17:43:44,717 - WARN  [Thread-37:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :2920, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:43:44,739 - WARN  [Thread-39:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:43:44,943 - WARN  [Thread-41:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :19, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[2;35m2016-03-30 17:44:45,615 - WARN  [Thread-2:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :3584, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:44:45,630 - WARN  [Thread-25:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:44:45,838 - WARN  [Thread-10:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :11, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2016-03-30 17:44:49,247 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x8a911711, /127.0.0.1:62955 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 17:44:49,247 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xce06c988, /127.0.0.1:62954 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 17:44:49,247 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xa33844d2, /127.0.0.1:62953 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2016-03-30 17:44:59,826 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2016-03-30 17:45:00,631 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2016-03-30 17:45:00,632 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2016-03-30 17:45:00,632 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2016-03-30 17:45:00,942 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2016-03-30 17:45:00,973 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2016-03-30 17:45:00,990 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[7;31m2016-03-30 17:45:02,011 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 17:45:02,027 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 17:45:02,027 - ERROR [main:XMemcachedClient@593] - connect to 127.0.0.1:11211 error
[mjava.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: no further information
	at com.google.code.yanf4j.core.impl.FutureImpl.get(FutureImpl.java:145)
	at net.rubyeye.xmemcached.XMemcachedClient.connect(XMemcachedClient.java:582)
	at net.rubyeye.xmemcached.XMemcachedClient.<init>(XMemcachedClient.java:870)
	at net.rubyeye.xmemcached.XMemcachedClientBuilder.build(XMemcachedClientBuilder.java:340)
	at com.juice.orange.game.cached.MemcachedResource.buildMemcached(MemcachedResource.java:34)
	at com.klw.fastfun.pay.data.ServerBootstrap.main(ServerBootstrap.java:68)
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:404)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 17:45:02,027 - ERROR [main:XMemcachedClient@627] - Connect to 127.0.0.1:11211 fail
[mjava.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: no further information
	at com.google.code.yanf4j.core.impl.FutureImpl.get(FutureImpl.java:145)
	at net.rubyeye.xmemcached.XMemcachedClient.connect(XMemcachedClient.java:582)
	at net.rubyeye.xmemcached.XMemcachedClient.<init>(XMemcachedClient.java:870)
	at net.rubyeye.xmemcached.XMemcachedClientBuilder.build(XMemcachedClientBuilder.java:340)
	at com.juice.orange.game.cached.MemcachedResource.buildMemcached(MemcachedResource.java:34)
	at com.klw.fastfun.pay.data.ServerBootstrap.main(ServerBootstrap.java:68)
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:404)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;32m2016-03-30 17:45:02,045 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2016-03-30 17:45:02,063 - INFO  [main:ServerBootstrap@129] - Application PID:	6548
[m[2;35m2016-03-30 17:45:04,046 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[2;35m2016-03-30 17:45:04,051 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2016-03-30 17:45:08,351 - WARN  [Thread-33:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :153, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:45:08,384 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :8, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:45:08,839 - WARN  [Thread-34:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :9, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2016-03-30 17:49:44,123 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x7ae73dde, /127.0.0.1:63109 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 17:49:44,139 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x0555b01b, /127.0.0.1:63108 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 17:49:44,139 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x35ac887e, /127.0.0.1:63110 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2016-03-30 17:53:39,255 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2016-03-30 17:53:39,879 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2016-03-30 17:53:39,879 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2016-03-30 17:53:39,879 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2016-03-30 17:53:40,145 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2016-03-30 17:53:40,194 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2016-03-30 17:53:40,194 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2016-03-30 17:53:40,210 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2016-03-30 17:53:40,210 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2016-03-30 17:53:40,233 - INFO  [main:ServerBootstrap@129] - Application PID:	6620
[m[2;35m2016-03-30 17:53:46,615 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :45, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 17:53:46,672 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :28, action:com.klw.fastfun.pay.common.action.CommonAction, method:getCustomInfo
[m[2;35m2016-03-30 17:53:48,609 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :889, action:com.klw.fastfun.pay.common.action.CommonAction, method:queryFilterBySpid
[m[2;35m2016-03-30 17:53:48,681 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[2;35m2016-03-30 18:04:05,407 - WARN  [Thread-34:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :2, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[7;31m2016-03-30 18:04:05,437 - ERROR [Thread-24:ConnectionResource@40] - insert into t_order_req_03 (ff_id, cp_id, imsi, imei, iccid, mobile, sp_id, pmodel, osversion, nwtype, ctech, fee, ip, province, cp_param, is_syn, is_success, pck,  sdk_ver,create_time, update_time) values(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,now(),now())
[mcom.mysql.jdbc.exceptions.MySQLIntegrityConstraintViolationException: Duplicate entry '03oox81778231688' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:931)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1631)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1723)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3256)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1313)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1585)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1500)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1485)
	at com.jolbox.bonecp.PreparedStatementHandle.executeUpdate(PreparedStatementHandle.java:203)
	at com.juice.orange.game.database.ConnectionResource.saveOrUpdate(ConnectionResource.java:38)
	at com.klw.fastfun.pay.data.dao.OrderReqDAO.addOrderReqInfo(OrderReqDAO.java:82)
	at com.klw.fastfun.pay.data.ao.OrderReqAO.addOrderReqInfo(OrderReqAO.java:29)
	at com.klw.fastfun.pay.common.action.impl.CommonActionImpl.addOrderReqInfo(CommonActionImpl.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.juice.orange.game.rmi.control.RPCServerControl.nextHandler(RPCServerControl.java:57)
	at com.juice.orange.game.util.threadpool.JuiceThreadPool$JuiceWorkThread.run(JuiceThreadPool.java:70)
[2;35m2016-03-30 18:04:05,444 - WARN  [Thread-24:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :28, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2016-03-30 18:05:10,359 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x49f2126b, /127.0.0.1:63378 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 18:05:10,365 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x71dfb65b, /127.0.0.1:63379 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2016-03-30 18:05:10,365 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x68dbb282, /127.0.0.1:63380 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2016-03-30 18:05:23,023 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://120.24.88.90:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2016-03-30 18:05:23,856 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2016-03-30 18:05:23,856 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2016-03-30 18:05:23,856 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2016-03-30 18:05:24,297 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2016-03-30 18:05:24,375 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2016-03-30 18:05:24,406 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2016-03-30 18:05:24,435 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2016-03-30 18:05:24,438 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2016-03-30 18:05:24,511 - INFO  [main:ServerBootstrap@129] - Application PID:	5984
[m[7;31m2016-03-30 18:05:24,734 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2016-03-30 18:05:24,765 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2016-03-30 18:05:26,774 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[7;31m2016-03-30 18:05:27,803 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 18:05:27,803 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 18:05:27,803 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2016-03-30 18:05:29,821 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 2 times
[m[2;35m2016-03-30 18:05:29,821 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2016-03-30 18:06:01,001 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :23979, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 18:06:07,954 - WARN  [Thread-8:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :0, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 18:06:08,030 - WARN  [Thread-10:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :22, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2016-03-30 18:06:17,877 - ERROR [Xmemcached-Reactor-3:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2016-03-30 18:06:17,877 - WARN  [Xmemcached-Reactor-3:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2016-03-30 18:06:19,881 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[7;31m2016-03-30 18:06:20,921 - ERROR [Xmemcached-Reactor-0:MemcachedConnector$InnerControllerStateListener@558] - Exception occured in controller
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 18:06:20,921 - ERROR [Xmemcached-Reactor-0:Reactor@325] - Reactor dispatch events error
[mjava.io.IOException: Connect to 127.0.0.1:11211 fail,Connection refused: no further information
	at net.rubyeye.xmemcached.impl.MemcachedConnector.onConnect(MemcachedConnector.java:423)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:303)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[7;31m2016-03-30 18:06:20,921 - ERROR [Heal-Session-Thread:MemcachedConnector$SessionMonitor@169] - Reconnect to 127.0.0.1:11211 fail
[m[2;35m2016-03-30 18:06:22,928 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 2 times
[m[2;35m2016-03-30 18:06:22,928 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2016-03-30 18:06:30,101 - WARN  [Thread-11:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :3236, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 18:06:30,133 - WARN  [Thread-14:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :12, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[2;35m2016-03-30 18:07:11,249 - WARN  [Thread-16:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :0, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m[2;35m2016-03-30 18:12:50,765 - WARN  [Thread-33:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPayCodes
[m